diff --git a/node_modules/path-to-regexp/dist/index.js b/node_modules/path-to-regexp/dist/index.js
index 49694bd..aa291da 100644
--- a/node_modules/path-to-regexp/dist/index.js
+++ b/node_modules/path-to-regexp/dist/index.js
@@ -69,9 +69,9 @@ function* lexer(str) {
                 throw new TypeError(`Unterminated quote at ${pos}: ${DEBUG_URL}`);
             }
         }
-        if (!value) {
-            throw new TypeError(`Missing parameter name at ${i}: ${DEBUG_URL}`);
-        }
+        // if (!value) {
+        //     throw new TypeError(`Missing parameter name at ${i}: ${DEBUG_URL}`);
+        // }
         return value;
     }
     while (i < chars.length) {
